### PR TITLE
Port knative target CRD updates

### PR DESCRIPTION
**Pull request by cab105**
Port updates made to the Knative Targets CRDs to enable `kubectl explain` to provide details on the spec attributes expected.

### CODE REVIEW

**Review by JeffNeff — APPROVED** [review]

File: config/300-awscomprehend-crd.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 TriggerMesh Inc.
+# Copyright (c) 2021 TriggerMesh Inc.
```
Comment (antoineco) [inline config/300-awscomprehend-crd.yaml:1]: Shouldn't be changed

**Review by antoineco — CHANGES_REQUESTED** [review]

File: config/300-googlefirestore-crd.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 TriggerMesh Inc.
+# Copyright (c) 2021 TriggerMesh Inc.
```
Comment (antoineco) [inline config/300-googlefirestore-crd.yaml:1]: Ditto

File: pkg/apis/targets/v1alpha1/alibabaoss_types.go [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 TriggerMesh Inc.
+Copyright (c) 2021 TriggerMesh Inc.
```
Comment (antoineco) [inline pkg/apis/targets/v1alpha1/alibabaoss_types.go:2]: Ditto

File: pkg/apis/targets/v1alpha1/aws_comprehend_types.go [hunk 4]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 TriggerMesh Inc.
+Copyright (c) 2021 TriggerMesh Inc.
```
Comment (antoineco) [inline pkg/apis/targets/v1alpha1/aws_comprehend_types.go:2]: Ditto

File: pkg/apis/targets/v1alpha1/elasticsearch_types.go [hunk 5]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 TriggerMesh Inc.
+Copyright (c) 2021 TriggerMesh Inc.
```
Comment (antoineco) [inline pkg/apis/targets/v1alpha1/elasticsearch_types.go:2]: Ditto

File: pkg/apis/targets/v1alpha1/hasura_types.go [hunk 6]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 TriggerMesh Inc.
+Copyright (c) 2021 TriggerMesh Inc.
```
Comment (antoineco) [inline pkg/apis/targets/v1alpha1/hasura_types.go:2]: Ditto

File: pkg/apis/targets/v1alpha1/infra_types.go [hunk 7]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 TriggerMesh Inc.
+Copyright (c) 2021 TriggerMesh Inc.
```
Comment (antoineco) [inline pkg/apis/targets/v1alpha1/infra_types.go:2]: Ditto

File: pkg/apis/targets/v1alpha1/jira_types.go [hunk 8]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 TriggerMesh Inc.
+Copyright (c) 2021 TriggerMesh Inc.
```
Comment (antoineco) [inline pkg/apis/targets/v1alpha1/jira_types.go:2]: Ditto

File: pkg/apis/targets/v1alpha1/oracle_types.go [hunk 9]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 TriggerMesh Inc.
+Copyright (c) 2021 TriggerMesh Inc.
```
Comment (antoineco) [inline pkg/apis/targets/v1alpha1/oracle_types.go:2]: Ditto

File: pkg/apis/targets/v1alpha1/salesforce_types.go [hunk 10]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 TriggerMesh Inc.
+Copyright (c) 2021 TriggerMesh Inc.
```
Comment (antoineco) [inline pkg/apis/targets/v1alpha1/salesforce_types.go:2]: Ditto

File: pkg/apis/targets/v1alpha1/uipath_types.go [hunk 11]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 TriggerMesh Inc.
+Copyright (c) 2021 TriggerMesh Inc.
```
Comment (antoineco) [inline pkg/apis/targets/v1alpha1/uipath_types.go:2]: Ditto

File: pkg/apis/targets/v1alpha1/zendesk_types.go [hunk 12]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 TriggerMesh Inc.
+Copyright (c) 2021 TriggerMesh Inc.
```
Comment (antoineco) [inline pkg/apis/targets/v1alpha1/zendesk_types.go:2]: Ditto

**Review by antoineco — CHANGES_REQUESTED** [review]

**Comment by antoineco** [comment]
`sed -i 's@Copyright (c)@Copyright@g' **/*.*` 

should be able to cleanly undo the changes to license headers.

**Comment by cab105** [comment]
> 
> 
> `sed -i 's@Copyright (c)@Copyright@g' **/*.*`
> 
> should be able to cleanly undo the changes to license headers.

I'm almost getting the vibe your flexing your sed like powers.

**Comment by antoineco** [comment]
How do you think I got all those files to finally use the same license header 😆

**Review by antoineco — APPROVED** [review]

**Pull request merged**